src/utils/comparison.rs

```diff
@@ -10,17 +11,30 @@ pub enum Comparison {
     NotEqual,
 }
 
-impl Not for Comparison {
+impl ComparisonOperator {
+    pub(crate) fn is_true(&self, vaf_a: AlleleFreq, vaf_b: AlleleFreq) -> bool {
+        match self {
+            ComparisonOperator::Equal => relative_eq!(*vaf_a, *vaf_b),
```

Comment:
Since we know the resolution of the universe, we can probably use a different (usually larger) epsilon for `relative_eq!` than the default.